docs/network_firewall.md

```diff
@@ -16,72 +16,83 @@ Network Firewall Setup Guide
-
-### SecureDrop-specific configuration
+### Assign interfaces
+
+Section 3.2.3, "Assigning Interfaces", of the pfSense Guide. Some firewalls, like the Netgate recommended in the Hardware Guide, have this set up already, in which case you can skip this step.
+
+### Initial configuration
+
+We will use the pfSense WebGUI to do the initial configuration of the network firewall.
+
+#### Connect to the pfSense WebGUI
+
+1. Boot the Admin Workstation into Tails from the Admin Live USB.
+2. Connect the Admin Workstation to the switch on the LAN. After a few seconds, you should see a pop up notification saying "Connection established" in the top right corner of the screen.
+3. Launch the Tor Browser, *Applications → Internet → Tor Browser*.
+    1. If there is a conflict between your WAN and the default LAN, you will not be able to connect to Tor, and may see a dialog that says "Tor is not ready. Start Tor Browser anyway?". Click "Start Tor Browser" to continue.
+5. Navigate to the pfSense GUI in the Tor Browser: https://192.168.1.1
+6. The firewall uses a self-signed certificate, so you will see a "This Connection Is Untrusted" warning when you connect. This is expected (see Section 4.5.6 of the pfSense Guide). You can safely continue by clicking "I Understand the Risks", "Add Exception...", and "Confirm Security Exception."
+7. You should see the login page for the pfSense GUI. Log in with the default username and password (admin / pfsense).
+
+#### Setup Wizard
+
+If you're setting up a brand new (or recently factory reset) router, pfSense will start you on the Setup Wizard. Click next, then next again. Don't sign up for a pfSense Gold subscription.
+
+On the "General Information" page, we recommend leaving your hostname as the default (pfSense). There is no relevant domain for SecureDrop, so we recommend setting this to "securedrop.local" or something similar. Use whatever DNS servers you wish. If you don't know what DNS servers to use, we recommend using Google's DNS servers: `8.8.8.8` and `8.8.4.4`. Click Next.
+
+Leave the defaults for "Time Server Information". Click Next.
+
+On "Configure WAN Interface", enter the appropriate configuration for your network. Consult your local sysadmin if you are unsure what to enter here. For many environments, the default of DHCP will work and the rest of the fields can be left blank. Click Next.
+
+For "Configure LAN Interface", set the IP address and subnet mask of the Application Subnet for the LAN interface. Click Next.
+
+Set a strong admin password. Click Next.
```

Comment:
Would rec changing the note about the firewall admin password to ref using keepass on the admin's networked tails workstation to generate a random password and using that instead of user created one.